### PR TITLE
docs: w3up-client README replace links to old tsdoc with links to correct/new tsdoc from monorepo

### PR DESCRIPTION
**Pull request by gobengo**
branchname s/trip/trim/

Motivation:
* https://github.com/web3-storage/w3up/issues/1215

### CODE REVIEW

File: packages/w3up-client/README.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ await client.setCurrentSpace(space.did())
 
 #### Delegating from Space to Agent
 
-In order to store data with w3up, your Agent will need a delegation from a Space. This automatically happens if you called `login(email)` then `registerSpace()`. However, if you are initializing the client with a previously created Space, you can `login(email)` then claim a delegation granted to the account associated with your email:
+In order to store data with w3up, your Agent will need a delegation from a Space. This automatically happens if you called `Account#provision` on a space. However, if you are initializing the client with a previously created Space, you can `login(email)` then claim a delegation granted to the account associated with your email:
```
Comment (alanshaw) [inline packages/w3up-client/README.md:159]: I think login does this automatically - i.e. no need to call `await capability.access.claim()`

**Review by alanshaw — APPROVED** [review]
LGTM - just a small tweak needed...

File: packages/w3up-client/README.md (hunk 1 above)
Comment (gobengo) [inline packages/w3up-client/README.md:159]:
```suggestion
In order to store data with w3up, your Agent will need a delegation from a Space. This automatically happens if you called `createSpace`. However, if you are initializing the client with a previously created Space, you can `login(email)` then claim a delegation granted to the account associated with your email:
```

Comment (gobengo) [reply packages/w3up-client/README.md]: https://github.com/web3-storage/w3up/pull/1218/commits/6c4262793307ea72420ea623a4da16cb67eff26d
https://github.com/web3-storage/w3up/pull/1218/commits/3fd0800fa6c9f1db1bbc895daa0bd2432f2357e7

**Pull request merged**